onedal/datatypes/_data_conversion.py

```diff
@@ -76,3 +75,19 @@ def func(x):
             return x
 
         return _apply_and_pass(func, *data)
+
+
+#TODO: Update with dpctl
+#For now it will fall back to numpy
+def _convert_one_to_dataframe(policy, x):
```

Comment:
I am not sure about `dataframe` in the func name.

Reply:
And also do we really need this func? We already have `_convert_to_supported`... that at least could be updated.

Reply:
`_convert_to_supported` - handles datatype, while `_convert_to_dataframe` should handle the following 3 options:
1) `list` as an input
2) `np.ndarray` as an input
3) `dpt.usm_ndarray` as an input

These things can be merged together but it will violate single responsibility principle.

Speaking about naming - I'm open to any suggestions. For me `dataframe` just sounds general enough to cover both ndarrays.